frontend/src/scenes/billing/CurrentPlan.tsx

```diff
@@ -43,10 +42,23 @@ export function CurrentPlan({ plan }: { plan: PlanInterface }): JSX.Element {
                             <h3 className="l3" style={{ marginBottom: 8 }}>
                                 {plan.name}
                             </h3>
-                            <Link target="_blank" to={`https://posthog.com/pricing#plan-${plan.key}?${UTM_TAGS}`}>
-                                More plan details <IconOpenInNew />
-                            </Link>
                             <div style={{ marginTop: 4 }}>{plan.price_string}</div>
+                            <div style={{ marginTop: 8, marginBottom: 8, alignItems: 'center', display: 'flex' }}>
+                                Set monthly billing limit to{' '}
+                                <InputNumber
+                                    style={{ width: 200, marginLeft: 8, marginRight: 8 }}
+                                    onChange={(value): void => {
+                                        if (billing) {
+                                            setBillingLimit({ ...billing, billing_limit: value })
+                                        }
+                                    }}
+                                    value={billing?.billing_limit || 0}
```

Comment:
This should probably be `defaultValue` as now it keeps changing back to 0 if no billing limit is set